var/spack/repos/builtin/packages/py-numpy/package.py

```diff
@@ -12,8 +12,7 @@ class PyNumpy(Package):
 
     extends('python')
     depends_on('py-nose')
-    depends_on('netlib-blas+fpic', when='+blas')
-    depends_on('netlib-lapack+shared', when='+blas')
+    depends_on('lapack+shared', when='+blas')
```

Comment:
Nope. Maybe it would if every lapack had a shared variant, but openblas doesn't. @alalazo will probably remove any changes to py-numpy, as discussed in #627.